src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -1,10 +1,11 @@
 ﻿using Microsoft.Diagnostics.Tracing.Parsers;
-using Microsoft.Diagnostics.Tracing.Session;
 using Reporting;
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Runtime.CompilerServices;
```

Comment:
What uses these new usings? Not seeing it in the code immediately..

Reply:
Yes it should be removed.